one-ui/src/test/java/com/marklogic/hub/oneui/AuthTest.java

```diff
@@ -34,6 +35,12 @@
     @Value("${test.mlHost:localhost}")
     public String mlHost;
 
+    @Value("${test.dataHubDeveloperUsername}")
+    public String mlDHDeveloper;
```

Comment:
Let's not come up with new names for these - just use "dataHubDeveloperUsername" and "dataHubDeveloperPassword".